modules/Emails/language/en_us.lang.php

```diff
@@ -370,5 +371,6 @@
     'LBL_MARK_READ' => 'Mark As Read',
     'LBL_MARK_FLAGGED' => 'Mark As Flagged',
     'LBL_MARK_UNFLAGGED' => 'Mark As Unflagged',
+    'LBL_OPT_IN_EMAIL_CREATED' => 'Opt In Email Sent'
```

Comment:
@daniel-samson ending comma not required here?

Reply:
Well, technically speaking no. it is not required. This is because it is the last element in the array ... but I will add it.